bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/PowerControlPower.java

```diff
@@ -46,13 +47,11 @@ public PowerControlPower(ConfigProvider configProvider, ScheduledExecutorService
 
     @Override
     public void onReceiveCommand(String channelId, LGWebOSHandler handler, Command command) {
+        final State state = handler.getSocket().getState();
         if (RefreshType.REFRESH == command) {
             handler.postUpdate(channelId, handler.getSocket().isConnected() ? OnOffType.ON : OnOffType.OFF);
         } else if (OnOffType.ON == command) {
-            if (handler.getSocket().isConnected()) {
-                logger.debug("Received ON - TV is already on.");
-                handler.postUpdate(channelId, OnOffType.ON);
-            } else {
+            if (State.DISCONNECTED == state) {
```

Comment:
I have to agree with this, you are already sending WOL packets even after it wakes up, so why would it matter sending it while it is disconnecting?

Reply:
the problem is not so much sending the WOL, the problem is, it will be ignored by the TV. It will still shut down, but then our item will be in an incorrect state, until the disconnect finally happens.

Reply:
I cxn certify that it is not in my case.
DISCONNECTING is a state while the TV is turned off but the websocket is still alive. So sending the WOL is turning on the TV and you receive at this time a new app value from the subscription and the binding is switching from DISCONNECTING to REGISTERED.

Reply:
the updated version will now send the WOL packet also in DISCONNECTING state. 
I will not have any effect if the TV is still shutting down, but it also has no negative effect. And since we are sending multiple packets with a bit of delay there is a chance that it works a few ms later.

Reply:
it should be resolved